### PR TITLE
clean up phpcr executor

### PR DESCRIPTION
**Pull request by dbu**
replace #284, without BC breaks.

### CODE REVIEW

File: lib/Doctrine/Common/DataFixtures/Purger/PHPCRPurger.php [hunk 1]
```diff
@@ -14,36 +14,20 @@
 class PHPCRPurger implements PurgerInterface
 {
     /**
-     * DocumentManager instance used for persistence.
-     * @var DocumentManager
+     * @var DocumentManagerInterface
```
Comment (Ocramius) [inline lib/Doctrine/Common/DataFixtures/Purger/PHPCRPurger.php]: @dbu in which version was this interface introduced? Should we add a `conflict` clause to `composer.json`?

Comment (dbu) [reply lib/Doctrine/Common/DataFixtures/Purger/PHPCRPurger.php]: it was added in 2013, was not aware that we had releases before that time :-D

its version 1.3.0, will add a conflicts

**Comment by dbu** [comment]
but the test failure came from .travis.yml not installing phpcr-odm at all :-O

there is no minimal version build

**Comment by dbu** [comment]
ok, now the build is green

**Comment by Ocramius** [comment]
Thanks @dbu!

**Pull request merged**